sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -79,30 +83,4 @@ public SdkMeter get(String instrumentationName, @Nullable String instrumentation
   public static SdkMeterProviderBuilder builder() {
     return new SdkMeterProviderBuilder();
   }
-
-  /**
-   * Register a view with the given {@link InstrumentSelector}.
-   *
-   * <p>Example on how to register a view:
-   *
-   * <pre>{@code
-   * // get a handle to the MeterSdkProvider
-   * MeterSdkProvider meterProvider = OpenTelemetrySdk.getMeterProvider();
-   *
-   * // create a selector to select which instruments to customize:
-   * InstrumentSelector instrumentSelector = InstrumentSelector.builder()
-   *   .setInstrumentType(InstrumentType.COUNTER)
-   *   .buildInstrument();
-   *
-   * // create a specification of how you want the metrics aggregated:
-   * AggregatorFactory aggregatorFactory = AggregatorFactory.minMaxSumCount();
-   *
-   * //register the view with the MeterSdkProvider
-   * meterProvider.registerView(instrumentSelector, View.builder()
-   *   .setAggregatorFactory(aggregatorFactory).build());
-   * }</pre>
-   */
-  public void registerView(InstrumentSelector selector, View view) {
```

Comment:
let's keep this, but deprecate it, and remove it in the next release, to provide a smoother transition for people who might already be using the existing views API directly.

Reply:
👌